.size-limit.js

```diff
@@ -0,0 +1,132 @@
+module.exports = [
+  {
+    "webpack": false,
+    "path": ["packages/koa-liveness-ping/dist"],
+    "limit": "1.25KB"
+  },
+  {
+    "webpack": false,
+    "path": ["packages/jest-dom-mocks/dist"],
+    "limit": "6.27KB"
+  },
+  {
+    "webpack": false,
```

Comment:
It'd be cool to add this file to our generators so that `yarn generate package` will include an entry here. Doesn't need to be part of this PR though.

Reply:
Sure. Will explore the generator for a bit and make a separate PR for that.